### ISSUE TITLE
How can I extract feature from Image

### ISSUE THREAD

**Issue by wting861006** [layumi/Person_reID_baseline_pytorch#112]
I found the function "extract_feature" in test.py.Does it can be used to feature comparison with image retrieval？
I test this extract feature cost time,it cost about 23/sec per image。
    # Extract feature
    with torch.no_grad():
        start=time.time()
        query_feature = extract_feature(model, dataloaders['query'])
        end=time.time()
        print(end-start)

**Comment by dyh127** [comment]
Hi, @wting861006,
The function 'extract_feature' is used only to extract features from images and cannot be used to compare feature.
Actually, the test.py is used to extract feature and save the feature and according labels. The comparison of features and the precision calculation is in evaluate.py. and the feature comparison uses cos distance, so just multiplying the gallery feature with query feature is OK.
